throttle/contract_tx_limiter_middleware.go

```diff
@@ -102,6 +150,11 @@ func NewContractTxLimiterMiddleware(cfg *ContractTxLimiterConfig,
 		next loomchain.TxHandlerFunc,
 		isCheckTx bool,
 	) (res loomchain.TxHandlerResult, err error) {
+		defer func(begin time.Time) {
+			lvs := []string{"method", "NewContractTxLimiterMiddleware", "error", fmt.Sprint(err != nil)}
+			contractTxMiddlewareExecutionLatency.With(lvs...).Observe(time.Since(begin).Seconds())
+		}(time.Now())
```

Comment:
This is going to include the time spent running the tx handler and other middleware, so not going to be very useful or tell us anything we don't know already from the other metrics, might as well remove it.

Reply:
Done.